app/controllers/application_controller.rb

```diff
@@ -97,6 +98,15 @@ def disable_caching
     response.headers['Pragma'] = 'no-cache'
   end
 
+  def cache_issuer_in_cookie
```

Comment:
This isn't encrypted at all, right?

Reply:
Correct

Reply:
Actually, looking at it, it appears that Rails is URI encoding it, but still not encrypted.

Reply:
URI encoding is probs fine